pkgs/by-name/go/godot_4/package.nix

```diff
@@ -112,17 +116,18 @@ stdenv.mkDerivation rec {
     pkg-config
   ];
 
-  nativeBuildInputs = [
-    autoPatchelfHook
-    installShellFiles
-    pkg-config
-    scons
-  ];
-
-  runtimeDependencies =
+  nativeBuildInputs =
     [
+      autoPatchelfHook
+      installShellFiles
+      pkg-config
+      scons
+      ### test
       alsa-lib
       libGL
+      vulkan-loader
+    ]
+    ++ lib.optionals withX11 [
```

Comment:
How is it used for **building** this package?
If a build tool is used for testing it should be allocated to `nativeCheckInputs`.

Runtime dependencies shouldn't be at `nativeBuildInputs`.